apps/api/src/models/session.py

```diff
@@ -1,11 +1,19 @@
 from datetime import UTC, datetime
+from enum import StrEnum, auto
 from uuid import UUID, uuid4
 
 from pydantic import BaseModel, Field
 
 
+class SessionStatus(StrEnum):
+    RUNNING     = auto()
+    FINISHED    = auto()
+    CANCELLED   = auto()
```

Comment:
I think using a pydantic model with Literal[] is better here for db compatibility

Reply:
The SessionStatus property "RUNNING" is interpreted as the string "running" by the interpreter during "compile time" according to (the documentation)[https://docs.python.org/3/library/enum.html#enum.StrEnum]. If the db compatibility is a concern we could remove auto and replace with a literal, but I think an enum is easier to maintain. 
(in this function)[https://github.com/Futino/aitino/blob/e44e543f832d3331e8a79af9b35ec069dee942a4/apps/api/src/interfaces/db.py#L114]